### DISCUSSION TITLE
Update delayed docs to show how you can create indirect deps using dask.graph_manipulation.bind

### DISCUSSION THREAD

**Discussion by ghost** [dask/dask#7963] [Ideas]
As proposed here: https://github.com/dask/dask/pull/7908 by @jsignell and @jrbourbeau - provide an example in the delayed documentation on how to create dependencies on tasks that don't take the upstream tasks result in the args.

---Start potential proposed doc addition--- (I'd imagine it would go at the end of `docs/source/delayed.rst`

Indirect Dependencies
---------------------

Sometimes you might find yourself wanting to add a dependency to a task that does
not take the result of that dependency as an input. For example when a task depends
on the side-effect of another task. In these cases you can use
``dask.graph_manipulation.bind``.

```
.. code-block:: python

    import dask
    from dask.graph_manipulation import bind

    DATA = []

    @dask.delayed
    def inc(x):
        return x + 1

    @dask.delayed
    def add_data(x):
        DATA.append(x)

    @dask.delayed
    def sum_data(x):
        return sum(DATA) + x

    a = inc(1)
    b = add_data(a)
    c = inc(3)
    d = add_data(c)
    e = inc(5)
    f = bind(sum_data, [b, d])(e)
    f.compute()
```

``sum_data`` will operate on DATA only after both the expected items have
been appended to it. ``bind`` can also be used along with direct dependencies
passed through the function arguments.

---End potential proposed doc addition---

**Comment by jsignell** [comment]
Please open a Pull Request on the file you mentioned. That is the best way to get direct feedback.
